docs/serverless/edr-manage/trusted-apps-ov.mdx

```diff
@@ -9,6 +9,10 @@ status: in review
 <DocBadge template="technical preview" />
 <div id="trusted-apps-ov"></div>
 
+<DocCallOut title="Note">
+To add Elastic Endpoint as a trusted application in your other antivirus (AV) software, refer to <DocLink slug="/serverless/security/allowlist-endpoint">Allowlist Elastic Endpoint in third-party antivirus apps</DocLink>.
```

Comment:
Same suggestion as in ESS about context-setting. Also, I think we can make the DocLink tag self-closing and won't need to include the link text, since it's the same as the target page's title.
```suggestion
If you use ((elastic-defend)) along with other antivirus (AV) software, you might need to configure the other system to trust ((elastic-endpoint)). Refer to <DocLink slug="/serverless/security/allowlist-endpoint" /> for more information.
```